pako/README.md

```diff
@@ -12,5 +12,24 @@ you can require the packaged library like so:
 
 ```clojure
 (ns application.core
-  (:require cljsjs.pako))
+  (:require cljsjs.pako :as pako))
```

Comment:
One learns something everyday, hopefully, thanks :-)
So do I need to do anything more or have you corrected this already and will merge ?